client/app/index.html

```diff
@@ -9,6 +9,7 @@
     <link rel="icon" type="image/png" sizes="32x32" href="./assets/images/favicon-32x32.png">
     <link rel="icon" type="image/png" sizes="96x96" href="./assets/images/favicon-96x96.png">
     <link rel="icon" type="image/png" sizes="16x16" href="./assets/images/favicon-16x16.png">
+    <link rel="icon" href="./assets/images/favicon.ico">
```

Comment:
Ho actually, there is a 404 error:

https://demo.redash.io/assets/images/favicon-16x16.png

thats why the favicon is not showing :/